src/compute/src/logging/persist.rs

```diff
@@ -18,6 +18,7 @@ use mz_storage_client::controller::CollectionMetadata;
 use crate::compute_state::ComputeState;
 
 pub(crate) fn persist_sink<G>(
+    scope: &G,
```

Comment:
Is there a reason why we need to pass the scope through everywhere? In other places we have just taken the one from a collection we pass around anyway: https://github.com/MaterializeInc/materialize/blob/b559b0e17f6e7a47565162d681c79a38ed23ccf1/src/compute/src/sink/persist_sink.rs#L90

Reply:
oh, I didn't think of taking it from a collection. It feels more straight forward to have it as an argument even if it exists as side information of a collection that also happens to be passed. I can remove it if you prefer the other way

Reply:
I don't mind either way, but I would suggest you replace the line I liked above to use the new `scope` argument as well, to have it consistent everywhere.